external/loaders/loaders/_config.py

```diff
@@ -56,6 +56,11 @@ def __post_init__(self):
 
 
 class BatchesLoader(abc.ABC):
+    """
+    Abstract base class for configuration classes that load batches. See
+    subclasses for concrete implementations you can use in configuration files.
+    """
+
```

Comment:
I tried to follow the "document on the class" because there is no init routine on those subclasses. https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html